flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TestingTaskExecutorGateway.java

```diff
@@ -140,6 +142,10 @@ public void releaseOrPromotePartitions(JobID jobId, Set<ResultPartitionID> parti
 		releaseOrPromotePartitionsConsumer.accept(jobId, partitionToRelease, partitionsToPromote);
 	}
 
+	@Override
+	public void releaseClusterPartitions(Collection<IntermediateDataSetID> dataSetsToRelease) {
+	}
```

Comment:
I think this method does not override a method as `releaseClsuterPartitions(Collection, Time)`.